src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9192,6 +9192,12 @@ public sealed partial class UnconditionalSuppressMessageAttribute : System.Attri
         public string? Scope { get { throw null; } set { } }
         public string? Target { get { throw null; } set { } }
     }
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
+    public sealed class ConstantExpectedAttribute : Attribute
```

Comment:
We follow alphabetical order in refs, therefore this should be added before `public static partial class Debug` type above